english/disadvantage_of_git

```diff
@@ -0,0 +1 @@
+Subproject commit 1018a704d5a7b8c6c202006be542f3feff2264a7
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.